Configuration/Eras/python/Modifier_run2_HI_2018_cff.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
+run2_HI_2018 = cms.Modifier()
```

Comment:
this should be renamed or removed, because the original submission was already replaced by ```customiseHybrid.runOnHybridZS```